tools/ci_build/github/azure-pipelines/mac-ios-packaging-pipeline.yml

```diff
@@ -195,6 +195,29 @@ jobs:
         "$(ORT_SHOULD_UPLOAD_ARCHIVES)"
     displayName: "Assemble artifacts"
 
+  - script: |
+      set -e -x
+      ls -R "$(Build.ArtifactStagingDirectory)"
+    displayName: "List staged artifacts"
+
+  # Package.swift needs to point to a file relative to it. Do a copy to simplify things
+  - script: |
+      cp "$(Build.ArtifactStagingDirectory)/pod-archive-onnxruntime-c-${ORT_POD_VERSION}.zip" swift/
```

Comment:
So, always add "set -x" to such multiple lines scripts.

Reply:
This is for test purposes? Local artefact is used for with in the pipelines right? 
Where prod release is pointing to the remote URL one. Do I understand this correctly?

Reply:
The CI will always use the local pod it just built. When we're running the CI on the release branch that file will eventually be deployed as the release, but that's a completely separate step that requires the CI to pass first. 

Package.swift will have to be updated once we have done the release to point to the new remote URL with correct checksum.